### PR TITLE
Update version_control.md

### PR DESCRIPTION
**Pull request by gperu**
Modified:
- how to cite
- credit
- hyperlinks

### CODE REVIEW

File: pages/your_tasks/version_control.md [hunk 1]
```diff
@@ -69,16 +69,19 @@ Implementing version control in a research context involves more than just choos
    * Periodically clean up old branches and review access permissions
 
 ## How to cite this page
-Giacomo Peru, TO-DO
+To be added.
+
+## Credit
+This page draws on widely accepted practices in version control and research software engineering. While not based on specific external sources, it represents common principles found in many educational resources in this field.
```
Comment (dgarijo) [inline pages/your_tasks/version_control.md:75]: such as...? I think this statement should be added when we have taken/reinterpreted text from another source. If it has not, then maybe not include it?

**Comment by dgarijo** [comment]
Ping @gperu on my comment

**Comment by gperu** [comment]
Hi @dgarijo I'm ok with your corrections. Sorry didn't didn't understand you were waiting for my reply.

**Comment by dgarijo** [comment]
@gperu no worries. In the interest of time I will merge, and you can do another PR to address my comment.

**Comment by gperu** [comment]
Please merge. I will address your comment later.

**Pull request merged**